fhir-install/src/main/docker/ibm-fhir-server/bootstrap.properties

```diff
@@ -0,0 +1,3 @@
+# disable writing to trace.log by only sending trace data to console
+com.ibm.ws.logging.trace.format=BASIC
+com.ibm.ws.logging.trace.file.name=stdout
```

Comment:
Question: bootstrap.properties is still overridden by the corresponding environment variables?  or does this take precedence?

Reply:
strangely, there are no corresponding environment variables defined by liberty for these.  instead, we defined our own and using those in our server.xml.   and the server.xml settings override the bootstrap properties.

Reply:
great